x-pack/legacy/plugins/lens/public/editor_frame_plugin/embeddable/embeddable_factory.ts

```diff
@@ -47,7 +43,10 @@ export class EmbeddableFactory extends AbstractEmbeddableFactory {
     });
     this.chrome = chrome;
     this.indexPatternService = indexPatternService;
-    this.expressionRenderer = expressionRenderer;
+  }
+
+  public setExpressionRenderer(renderer: ExpressionRenderer) {
```

Comment:
I don't love this. If the renderer is required for this object to be used, it should be part of the constructor.